### ISSUE TITLE
How works UserDataDir ?

### ISSUE THREAD

**Issue by FournyP** [chromedp/chromedp#798]
#### What versions are you running?

<pre>
chromedp: v0.6.10
chrome: 90.0.4430.85
golang version: 1.16.3 windows/amd64
</pre>

#### What did you do? Include clear steps.

I use chromedp to automate a process and I need chrome extensions. The https://github.com/chromedp/chromedp/issues/279 issue say that I need to open chrome with a specific user

So, I have search to launch chrome with a specific user and I have created a context like this:
```golang
opts := append(chromedp.DefaultExecAllocatorOptions[:],
	chromedp.Flag("headless", false), // To display the browser
	chromedp.Flag("disable-extensions", false),
	chromedp.UserDataDir(os.Getenv("CHROME_USER_DIR")),
)

ctx, cancel := chromedp.NewExecAllocator(context.Background(), opts...)
```

and "CHROME_USER_DIR" is init like this before with: 
```golang
// Chrome values
os.Setenv("CHROME_USER_DIR", "C:\\Users\\pierr\\AppData\\Local\\Google\\Chrome\\User Data\\Default")
```

After that, I launch chromedp like this: 
```golang
ctx, cancel := chromedp.NewContext(*client.ctxParent)
defer cancel()

//Execute the creation of account
err = chromedp.Run(ctx,
	chromedp.Navigate(REGISTER_ENDPOINT),
	chromedp.Sleep(10 * time.Second),
)
```

and the client.ctxParent contains the context created in the first step.

I want to know if it is possible to launch chrome with my account and their extensions.
What I'm doing wrong ?

I have see https://github.com/chromedp/chromedp/issues/110 issue but I don't have google-chrome command line.

#### What did you expect to see?
I want to see the browser open with my account and chrome extensions.

#### What did you see instead?
I see a guest account without chrome extensions

Have a good day and thanks you for your work.
If you want more informations, feel free to ask me

**Comment by ZekeLu** [comment]
Please check https://github.com/chromedp/chromedp/issues/764#issuecomment-809536531.

TL;DR, here is what you need (tested on Linux only):
```go
	opts := append(chromedp.DefaultExecAllocatorOptions[:],
		// if user-data-dir is set, chrome won't load the default profile,
		// even if it's set to the directory where the default profile is stored.
		// set it to empty to prevent chromedp from setting it to a temp directory.
		chromedp.UserDataDir(""),
		// in headless mode, chrome won't load the default profile.
		chromedp.Flag("headless", false),
		chromedp.Flag("disable-extensions", false),
	)
```

**Comment by FournyP** [comment]
Hello @ZekeLu,

It works on windows too, thanks a lot for your reply.

Have a good day
